.github/CODEOWNERS

```diff
@@ -1,10 +1,15 @@
 ################################################################################
 #                        IMPORTANT NOTES FOR CODEOWNERS:                       #
 #                                                                              #
-# - If you will be unavailable for more than 24 hours, please replace your     #
-#   ownership with a delegate, file an issue, and update your membership in    #
-#   your group's settings to exempt you from code reviews. Please make sure to #
-#   restore ownership after you're able to continue reviews.                   #
+# - If you will be unavailable for more than 24 hours, please do the all of    #
```

Comment:
"the all" --> "all"

Reply:
Done.

Reply:
I've verified this change has landed in the PR, so marking this as resolved since you've approved the PR @seanlip.